client-src/elements/chromedash-gate-column.js

```diff
@@ -462,12 +493,24 @@ export class ChromedashGateColumn extends LitElement {
   }
 
   renderQuestionnaireSkeleton() {
-    return nothing;
+    return html`
+      <h2>Survey questions</h2>
+      <div id="questionnaire">Loading...</div>
+      <p class="instructions">&nbsp;</p>
+    `;
   }
 
   renderQuestionnaire() {
-    // TODO(jrobbins): Implement questionnaires later.
-    return nothing;
+    const questionnaireText = GATE_QUESTIONNAIRES[this.gate.gate_type];
+    const instructions = (
+      questionnaireText ?
+        html`Please post responses in the comments below.` :
+        html`&nbsp`);
```

Comment:
```suggestion
        "&nbsp";
```
Same here?

Reply:
I think it needs it, otherwise the `&` will get escaped.

Reply:
But, you do make me realize that it needs a semicolon.